charmstore/fakeclient.go

```diff
@@ -421,7 +426,7 @@ func (r Repository) CharmInfo(charmURL string) (*charms.CharmInfo, error) {
 //
 // Part of the charmrepo.Interface
 func (r Repository) Resolve(ref *charm.URL) (canonRef *charm.URL, supportedSeries []string, err error) {
-	return r.addRevision(ref), []string{"trusty", "wily", "quantal"}, nil
+	return r.addRevision(ref), []string{"trusty", "wily", "quantal", "kubernetes"}, nil
```

Comment:
Can't have charms operating as both k8s and non k8s, or WIndows and Ubuntu etc
All series for a given charm need to be the same OS